great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -231,6 +231,10 @@ def _send_get_request_to_api(self, url: str, params: dict | None = None) -> dict
             raise StoreBackendError(  # noqa: TRY003
                 f"Unable to get object in GX Cloud Store Backend: {get_user_friendly_error_message(http_err)}"  # noqa: E501
             )
+        except requests.ConnectionError as conn_err:
+            raise StoreBackendError(  # noqa: TRY003
+                f"Unable to get object in GX Cloud Store Backend: {conn_err}"
+            )
```

Comment:
Maybe not the best practice but the Cloud fake raises this error if we hit an endpoint that isn't implemented. The `Store.add` flow makes a GET to check existence before POSTing and when it misses, it manifests as this error in our fake.

Reply:
I'll try adding a `from conn_err` here.